server/resp/src/main/java/org/infinispan/server/resp/commands/json/JSONAPPEND.java

```diff
@@ -44,7 +43,8 @@ static BiConsumer<List<Long>, ResponseWriter> newIntegerOrErrorWriter(byte[] pat
         // legacy path just one result and it must be not null
         return (c, writer) -> {
             if (c == null || c.size() == 0 || c.get(0) == null) {
-                throw new CacheException("Path '" + RespUtil.ascii(path) + "' does not exist or not a " + opType);
```

Comment:
We shouldn't have to change these or the others in this PR with the new changes to exception handling.